crmsh/corosync.py

```diff
@@ -57,6 +57,14 @@ def __repr__(self):
         return '%s: %s' % (self.key, self.value)
 
 
+class QDevice(object):
+    def __init__(self, ip, port=5403, algo="ffsplit", tie_breaker="lowest"):
+        self.ip = ip
+        self.port = port
+        self.algo = algo
+        self.tie_breaker = tie_breaker
```

Comment:
the tie_breaker should be:

auto_tie_breaker: 1
auto_tie_breaker_node: lowest

auto_tie_breaker = 1 mean to enable auto_tie_breaker, this option is disabled by default, we should enable it.
while auto_tie_breaker_node is the preference that qnetd to decide who should have the votes if there is brain split